examples++/echo_server.cpp

```diff
@@ -32,7 +32,7 @@ int main(void)
 	    srv.sndto(answer,from,fromport);
 	}
 
-	srv.destroy();
+	//srv.destroy(); //libsocket::inet_dgram_server has destructor
```

Comment:
`destroy()` is intended as explicit way to make a server stop. How about leaving this in (it's also part of the test suite after all) and writing it as

```
// libsocket::inet_dgram_server also has a destructor doing this for us.
srv.destroy();
```

Reply:
Many thanks for your answer, updated - I will study the lib as soon as I have time :), new to programming.